cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/ExportXmlCompare.java

```diff
@@ -615,7 +620,12 @@ static ComparisonResult ignoringRdfChildLookupStaticVarCompensator(Comparison co
     }
 
     static ComparisonResult ignoringRdfChildLookupRegulatingControl(Comparison comparison, ComparisonResult result) {
-        if (result == ComparisonResult.DIFFERENT && comparison.getType() == ComparisonType.CHILD_LOOKUP) {
+        if (result == ComparisonResult.DIFFERENT && comparison.getType() == ComparisonType.TEXT_VALUE) {
```

Comment:
behaviour of the method has changed, I suggest renaming it to `ignoringRegulatingControl`

Reply:
Done